rpcs3/Emu/Io/pad_config_types.h

```diff
@@ -8,6 +8,7 @@ enum class pad_handler
 	keyboard,
 	ds3,
 	ds4,
+	ds5,
```

Comment:
I'd call all of this dual_sense, DualSense, dual_sense_pad_handler etc...

Reply:
I was a little hesitant to call it dualsense because of length, or ds because that would be confusing. But i think I agree that dualsense in full would be less confusing than ds5

Reply:
I think it's actually fine if you keep all the ds5 references in code and the filenames.
Just fix the user accessible strings to say "DualSense" and it should be okay.

Reply:
For this name, after thinks, write dual_sense is good idea using ds5 cause strange things in futur ex for ps6 controller called DualSense 2, not sur using ds6 and etc lol

Reply:
What about dualsense_pad_handler, dualsenseDevice, DualSenseDevice? Otherwise, I have almost every front-end reference cleaned up to say DualSense.

Reply:
wait i show you how i thinks set :)

Reply:
there's no space in DualSense, so it should be dualsense